package.json

```diff
@@ -46,11 +46,15 @@
     "jest": "^24.9.0",
     "lint-staged": "9.5.*",
     "node-fetch": "2.3.*",
-    "nodemon": "^1.19.1"
+    "nodemon": "^1.19.1",
+    "rimraf": "^3.0.2"
   },
   "scripts": {
+    "build": "rimraf ./dist && babel ./src --out-dir ./dist --copy-files",
+    "build:debug": "rimraf ./dist && babel ./src --out-dir ./dist --copy-files --source-maps",
     "lint": "eslint src tests",
     "start": "nodemon --exec babel-node --watch src src/index.js",
+    "start:prod": "nodemon --exec node dist/index.js",
```

Comment:
does a prod launcher need nodemon? It seems to me like it should just compile (if needed) and run the code.

Reply:
No, it doesn't. The prod script can be executed with node directly, I will update the command.